include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -17,15 +17,14 @@
 #ifndef _ONEDPL_SERIAL_NUMERIC_IMPL_H
 #define _ONEDPL_SERIAL_NUMERIC_IMPL_H
 
-#if (__cplusplus >= 201703L)
-#    include <functional>
-#    include <iterator>
-#    include <numeric>
+#include <functional>
+#include <iterator>
+#include <numeric>
 namespace oneapi
 {
 namespace dpl
 {
-#    if _ONEDPL_HAS_NUMERIC_SERIAL_IMPL
+#if _ONEDPL_HAS_NUMERIC_SERIAL_IMPL
```

Comment:
Is there any C++ library with doesn't have `::std::reduce` and that library is compiled with "std=C++17"?
If there is not, that I think that we should remove 27-52 lines as well.